### DISCUSSION TITLE
Find when app last crashed (non zero exit code)

### DISCUSSION THREAD

**Discussion by mattbdc** [dokku/dokku#5370] [General]
With the default policy of        Ps restart policy:             on-failure:10 how can I see when dokku last restarted it / and ideally the logs immediately prior to that, or even the up time of the current running restart, or the times of restarts. ps:report doesn't give me that info

**Comment by josegonzalez** [comment]
You should be able to get this information if you tail the docker event stream. We also run a `dokku-event-listener`, which may log when a container restarts as it needs that info to know when to regenerate nginx config.
